sample/Assets/Scripts/SelectAuthMethodScript.cs

```diff
@@ -53,7 +53,20 @@ public void UseDeviceCodeAuth()
     public void UsePKCE()
     {
         SampleAppManager.UsePKCE = true;
+#if UNITY_WEBGL
+        string url = Application.absoluteURL;
+        Uri uri = new Uri(url);
+        string scheme = uri.Scheme;
+        string hostWithPort = uri.IsDefaultPort ? uri.Host : $"{uri.Host}:{uri.Port}";
+        string fullPath = uri.AbsolutePath.EndsWith("/") ? uri.AbsolutePath : uri.AbsolutePath.Substring(0, uri.AbsolutePath.LastIndexOf('/') + 1);
+
+        string redirectUri = $"{scheme}://{hostWithPort}{fullPath}callback.html";
+        string logoutRedirectUri = $"{scheme}://{hostWithPort}{fullPath}logout.html";

```

Comment:
Does this mean, the customers have to set these redirect and logout redirect URIs in Hub?
e.g.
A redirect URI that ends in `XX://XXX:XXX/callback.html`
A logout redirect URI that ends in `XX://XXX:XXX/logout.html`

Reply:
yes they will need to set them in Hub